### PR TITLE
Add doc for Auth0 as KM

### PR DESCRIPTION
**Pull request by ChamodDamitha**
Moved the documentation from https://github.com/wso2-extensions/apim-km-auth0/blob/master/docs/config.md

### CODE REVIEW

**Review by tharindu1st — APPROVED** [review]

File: en/docs/administer/key-managers/configure-auth0-connector.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+    <a name="step13"></a>
+    
+4.  Create a new scope named `default` for the API by navigating to permissions tab in the custom API.    
+
+    [![auth0 create scope]({{base_path}}/assets/img/administer/auth0-create-scope.png)]({{base_path}}/assets/img/administer/auth0-create-scope.png)
+
+## Step 2 - Configure WSO2 API Manager
+
+1.  Log into the admin portal of the API Manager and add a new Key Manager.
```
Comment (isharac) [inline en/docs/administer/key-managers/configure-auth0-connector.md:38]: admin portal or Admin Portal? @Mariangela wdyt?

File: en/docs/administer/key-managers/configure-auth0-connector.md [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+    
+    [![auth0 add app admin]({{base_path}}/assets/img/administer/auth0-add-app-admin.png)]({{base_path}}/assets/img/administer/auth0-add-app-admin.png)
+    
+2.  Then select the Key manager type as Auth0 and provide the relevant fields accordingly.
```
Comment (isharac) [inline en/docs/administer/key-managers/configure-auth0-connector.md:42]: Kry Manager ? @Mariangela

File: en/docs/administer/key-managers/configure-auth0-connector.md [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+    set its value as <code>none</code>. Select the token validation method as <code>Self validate JWT</code> like below.
+    
+    [![auth0 token validation]({{base_path}}/assets/img/administer/auth0-token-validation.png)]({{base_path}}/assets/img/administer/auth0-token-validation.png)    
+
+3.  Set the grant types which are allowed in Auth0 as below. The format of the grant types can be found in 
+<a href="https://auth0.com/docs/applications/application-grant-types">this link</a>.
+    
+    [![auth0 grant types]({{base_path}}/assets/img/administer/auth0-grant-types.png)]({{base_path}}/assets/img/administer/auth0-grant-types.png)        
+    
+4.  The client ID, Client secret of the application created to invoke Manage API should be provided for the settings. You can get to the audience
```
Comment (isharac) [inline en/docs/administer/key-managers/configure-auth0-connector.md:62]: Client Secret

**Review by isharac — DISMISSED** [review]

File: en/docs/administer/key-managers/configure-auth0-connector.md (hunk 1 above)
Comment (Mariangela) [reply en/docs/administer/key-managers/configure-auth0-connector.md:38]: Admin Portal

**Review by Mariangela — APPROVED** [review]

**Pull request merged**